services/terrain/src/terrain/clients/data_info/raw.clj

```diff
@@ -200,20 +201,25 @@
 (defn set-avus
   "Set the iRODs AVUs to a specific set."
   [user path-uuid avu-map]
-  (request :put ["data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :post ["data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
 
 (defn add-avus
   "Add AVUs to a data item."
   [user path-uuid avu-map]
-  (request :post ["data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :put ["data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
+
+(defn metadata-copy
+  [user force path-uuid copy-request]
+  (request :post ["data" path-uuid "avus" "copy"]
+           (mk-req-map user (json/encode copy-request) {:force force})))
 
 (defn admin-add-avus
   "Add AVUs, allowing administrative AVUs to be included, for a data item."
   [user path-uuid avu-map]
-  (request :post ["admin" "data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :put ["admin" "data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
```

Comment:
The metadata endpoints were already using `POST` requests for `set` operations and `PUT` for `add/update`, so I just updated the data-info endpoints to be consistent with those.